airbyte-cdk/python/airbyte_cdk/sources/file_based/file_based_source.py

```diff
@@ -217,11 +223,15 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
                         CursorField(DefaultFileBasedStream.ab_last_mod_col),
                     )
                     stream = FileBasedStreamFacade.create_from_stream(
-                        self._make_default_stream(stream_config, cursor), self, self.logger, stream_state, cursor
+                        self._make_default_stream(stream_config, cursor, self._use_file_transfer(parsed_config)),
+                        self,
+                        self.logger,
+                        stream_state,
+                        cursor,
```

Comment:
Nit. This is a lot of positional args. Maybe worth switching to kwarg syntax here for readability? (Both for `create_from_stream()` and `_make_defualt_stream()`.

Reply:
switching to kwarg syntax here for readability ✅